lib/client/client.js

```diff
@@ -1150,7 +1150,7 @@ Client.prototype._sendSocket = function _sendSocket (message,
   var sentEmitter = false
 
   function sendResult (event, obj) {
-    if (event === 'error' && self.listeners('resultError')) {
+    if (event === 'error') {
```

Comment:
```suggestion
    if (event === 'error' && self.listenerCount('resultError')) {
```

Reply:
Seems unnecessary. If there are no listeners, the emit is effectually a noop. It is not custom to look for listeners before emitting an event and it is not done anywhere else in this code base.

Reply:
Makes sense. I was only going on the intent implied by the original code in this change set.